db/schema.rb

```diff
@@ -10,7 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2024_06_06_063306) do
+ActiveRecord::Schema[7.1].define(version: 2024_06_10_052659) do
+  create_table "courses", charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
+    t.string "title"
+    t.string "technologies"
+    t.integer "duration"
+    t.text "table_of_contents"
+    t.bigint "tutor_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "free_or_paid"
```

Comment:
use rails `enum` for this https://api.rubyonrails.org/v5.1/classes/ActiveRecord/Enum.html